prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -66,7 +65,11 @@ def omen_replicate_from_tx(
         10 if market_type == MarketType.POLYMARKET else 1000,
         market_type,
         filter_by=FilterBy.OPEN,
-        sort_by=sort_by,
+        sort_by=(
+            SortBy.NONE
+            if market_type == MarketType.POLYMARKET
+            else SortBy.CLOSING_SOONEST
```

Comment:
I noticed today that on Manifold, when replication form markets closing in less than a week, we have 6 markets available with default sorting or 460 markets with this sorting. So this gives more opportunities to actually go through our filters (is_predictable, is_predictable_without_description) and replicate something.